gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/MongoEventRepository.java

```diff
@@ -92,8 +91,9 @@ public Event update(Event event) throws TechnicalException {
             eventMongo.setCreatedAt(event.getUpdatedAt());
             eventMongo.setUpdatedAt(event.getUpdatedAt());
             eventMongo.setEnvironments(event.getEnvironments());
+            eventMongo.setOrganizations(event.getOrganizations());
```

Comment:
Shouldn't you use a mapper here instead of manually setting it?

Reply:
This is a special case because it retrieves the event from the DB and updates a selection of fields. I guess it's ok to do it this way for such a particular case. wdyt?